### PR TITLE
Fixes master

### PR DESCRIPTION
**Pull request by Cyberboss**
>Travis not catching this how?

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #if DM_VERSION >= 512
 #error HEY, PIXEL_SCALE probably exists now, remove this gross ass shim.
 #endif
+#endif
```
Comment (RemieRichards) [inline code/game/atoms_movable.dm]: now hey this doesn't match the reference at all.

Comment (RemieRichards) [inline code/game/atoms_movable.dm]: I know ```#endif```s aren't necessary in all cases, cos oranges did an 
```#if DM_VERSION > 512
...
#elseif
```

with no ```#endif```, the reference also says ```#if``` goes to the next ```#elseif,#endif```, which implies they don't need to match amounts.

Comment (Cyberboss) [inline code/game/atoms_movable.dm]: Doesn't compile on 511.1378

**Comment by lzimann** [comment]
thanks byond

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/atoms_movable.dm]: stonerrrrrrrrrrrrrrr! @MrStonedOne

Comment (MrStonedOne) [inline code/game/atoms_movable.dm]: why you blame me?

Comment (MrStonedOne) [inline code/game/atoms_movable.dm]: also, link me to oranges thing he did, i want to see what you are talking about

Comment (RemieRichards) [inline code/game/atoms_movable.dm]: wasn't blame, was asking your opinion.
https://github.com/tgstation/tgstation/blob/9e1ef0ffe2a7d955a40fd2d83382785353a29cb4/code/__HELPERS/_lists.dm#L452

Comment (Cyberboss) [inline code/game/atoms_movable.dm]: > byond

Comment (RemieRichards) [inline code/game/atoms_movable.dm]: So it just looks instead like the ref is shit, not explicitly saying that ```#endif```s are required, instead suggesting that any ```#else```,```#elseif```, or ```#endif``` is enough (and certainly not stating that one is require per instance)

Comment (MrStonedOne) [inline code/game/atoms_movable.dm]: oh god.... so what thats doing is embedding the rest of the entire project in the else if

meaning shit will break massively once we try to use 512

Comment (RemieRichards) [inline code/game/atoms_movable.dm]: I did consider that, but assumed no.
hilarious.

Comment (lzimann) [inline code/game/atoms_movable.dm]: Well, at least we will know what it is breaking!

**Pull request merged**